r/DESCRIPTION

```diff
@@ -0,0 +1,22 @@
+Package: rrrow
+Title: Bindings to 'Apache' 'Arrow'
+Version: 0.0.0.9000
+Authors@R: c(
+    person("Romain", "François", email = "romain@purrple.cat", role = c("aut", "cre"))
```

Comment:
This should have Apache Arrow / dev@arrow.apache.org in the metadata somewhere -- it looks like SparkR and some other projects cite authors other than the Apache project https://cran.r-project.org/web/packages/SparkR/index.html

Reply:
done, added the ASF as an author and copyright holder